src/data/markdown/docs/20 jslib/01 jslib/05 aws/KMSClient/98 KMSDataKey.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'KMSDataKey'
+description: 'KMSDataKey is returned by the KMSClient.*DataKey methods that query KMS data keys'
+excerpt: 'KMSDataKey is returned by the KMSClient.*DataKey methods that query KMS data keys'
+---
+
+KMSClient.*DataKey methods, querying Key Management Service data keys, return some KMSDataKey instances. The KMSDataKey object describes an Amazon Key Management Service data key. For instance, the [`generateDataKey`](/javascript-api/jslib/aws/kmsclient/kmsclient-generate-data-key/) returns the generated KMSDataKey object.   
```

Comment:
```suggestion
`KMSClient.*DataKey` methods, querying Key Management Service data keys, return some KMSDataKey instances. The KMSDataKey object describes an Amazon Key Management Service data key. For instance, the [`generateDataKey`](/javascript-api/jslib/aws/kmsclient/kmsclient-generate-data-key/) returns the generated KMSDataKey object.   
```

I assume that's a wildcard, which means it could be in a terminal command or code file, which means it should be in `monospace`font.

Reply:
I'm really happy to drop "allow" all over. Thanks for the recommendation, will integrate it indeed 🙏🏻